src/filters/PaddingSides.ts

```diff
@@ -0,0 +1,164 @@
+import type { Filter } from './Filter';
+
+/**
+ * class used for paddings of text styles and filters
+ * @category filters
+ * @advanced
+ */
+export class PaddingSides
```

Comment:
Can this not be a class? The usages below are mostly normalizing a value or doing some compute. This can be function and doesn't add much value as a class.